src/mame/drivers/nmk16.cpp

```diff
@@ -4105,6 +4105,9 @@ void nmk16_state::set_hacky_screen_lowres(machine_config &config)
 	m_screen->set_size(256, 256);
 	m_screen->set_visarea(0*8, 32*8-1, 2*8, 30*8-1);
 	m_screen->set_palette(m_palette);
+
+	NMK_16BIT_SPRITE(config, m_spritegen, 384 * 263);
```

Comment:
How are you getting the `384 * 263` value to use for the clock?  Shouldn't it be the pixel clock or the bus clock for correct documentation?